lingpy3/algorithm/cython/_calign.py

```diff
@@ -165,7 +165,7 @@ def globalign(
             almB += [seqB[i-1]]
             i -= 1
             j -= 1
-        else:
+        else:  # pragma: no cover
```

Comment:
sorry, we can easily replace this line by 
```python
elif traceback[i][j] == 2
```
as there are only three numbers in the arrays. Is this the preferred way to do?